docs/labs/compose.rst

```diff
@@ -0,0 +1,90 @@
+.. _labs-compose:
+
+====================
+Flocker with Compose
+====================
+
+With the :ref:`Flocker Docker plugin <labs-docker-plugin>`, and with Compose support for the ``volume_driver`` field, you can use Flocker together with Docker Compose.
+
+First, you need to :ref:`install Flocker <labs-installer>` and the :ref:`Flocker Docker plugin <labs-docker-plugin>`.
+You can use our experimental  :ref:`Flocker Installer <labs-installer>` to do this.
+
+Then, you need a version of Compose that supports Flocker volumes.
+
+Run the following command to do this (this branch that will be installed corresponds to `this pull request <https://github.com/docker/compose/pull/1502>`_):
+
+.. prompt:: bash $
+
+    sudo pip install git+https://github.com/lukemarsden/compose.git@volume_driver
+
+docker-compose.yml
+==================
+
+To make use of Flocker volumes with Compose, you will need to specify a ``volume_driver`` field in your ``docker-compose.yml`` file.
+
+Here is an example of a simple application that has 2 containers ``web`` and ``redis``.  Notice how the ``redis`` container has a data volume and has the ``volume_driver`` field set to ``flocker``.
+
+.. code-block:: yaml
+
+    web:
+      image: binocarlos/moby-counter
```

Comment:
Having things in our docs tied to a particular employee is setting ourselves up for failure (e.g. if that employee leaves).